### PR TITLE
docs: How to exchange encrypted data initial commit

### PR DESCRIPTION
**Pull request by gkc**
**- What I did**
Create doc specifying how a client for one atSign can exchange encrypted data with another atSign

### CODE REVIEW

**Comment by gkc** [comment]
@cpswan Work in progress

**Comment by gkc** [comment]
@cpswan ready for review (diagrams still TODO)

File: usage-examples/how-to-exchange-encrypted-data.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    - Send `plookup:publickey@bob`
+    - Successful response `data:<bobsBase64EncodedPublicKey>`
+  - Encrypt $base64EncodedSharedAESKey with bobsBase64EncodedPublicKey and base64-encode the result =>
+    $base64EncodedEncryptedForBobSharedAESKey
+  - Store in the atServer
+    - Send `update:ttr:86400:@bob:shared_key@alice $base64EncodedEncryptedForBobSharedAESKey` and handle the 
+      response
+#### 4. Encrypt some data
+- Encrypt your $data with $sharedAESKey and base64-encode the result => $base64EncodedEncryptedData
+  - We currently use the CTR (aka SIC) mode of the AES cipher, with PKCS7Padding. Java example:
```
Comment (JeremyTubongbanua) [inline usage-examples/how-to-exchange-encrypted-data.md:108]: Maybe important to mention the initialization vector an array of 16 zeroes?

Comment (gkc) [reply usage-examples/how-to-exchange-encrypted-data.md:108]: I'd rather not, given the imminent release of the fix to use new IVs every time

**Review by cpswan — APPROVED** [review]

**Comment by cpswan** [comment]
As discussed earlier, getting this merged, and we can have another PR for diagrams etc.

**Pull request merged**